test/mix/tasks/cache.clean_test.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Mix.Tasks.Cache.CleanTest do
```

Comment:
This does not appear to be testing anything: https://codecov.io/gh/mbta/skate/compare/8c56a4beab5aeac5fb15c6fa060c057137e12124...7bd5f55fe020d2348fc4edc0c767f60c21ca0252/diff#D11-10

Reply:
The filename had the wrong extension so it wasn't being included in the test suite.